### PR TITLE
Interactive dashboard example

### PR DESCRIPTION
**Pull request by domoritz**
screen shot 2017-09-10 at 14 01 12

### CODE REVIEW

**Comment by kanitw** [comment]
The fact that the rect's color and the point's size encode the same thing is a bit weird.
I think using filled circles (or squares) with size for both layer might be better:

```
{
  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
  "description": "A dashboard with cross-highlighting.",
  "data": {"url": "data/movies.json"},
  "vconcat": [
    {
      "layer": [{
        "mark": "circle",
        "encoding": {
          "x": {
            "bin": {"maxbins": 10},
            "field": "IMDB_Rating",
            "type": "quantitative"
          },
          "y": {
            "bin": {"maxbins": 10},
            "field": "Rotten_Tomatoes_Rating",
            "type": "quantitative"
          },
          "size": {
            "aggregate": "count",
            "type": "quantitative",
            "legend": {
              "title": "All Movies"
            }
          },
          "opacity": {"value": "0.4"}
        }
      }, {
        "transform": [{
          "filter": {"selection": "pts"}
        }],
        "mark": "circle",
        "encoding": {
          "x": {
            "bin": {"maxbins": 10},
            "field": "IMDB_Rating",
            "type": "quantitative"
          },
          "y": {
            "bin": {"maxbins": 10},
            "field": "Rotten_Tomatoes_Rating",
            "type": "quantitative"
          },
          "size": {
            "aggregate": "count",
            "type": "quantitative",
            "legend": {
              "title": "In Selected Category"
            }
          }
        }
      }]
    }, {
      "width": 330,
      "height": 120,
      "mark": "bar",
      "selection": {
        "pts": {"type": "single", "encodings": ["x"]}
      },
      "encoding": {
        "x": {"field": "Major_Genre", "type": "nominal", "axis": {"labelAngle": -40}},
        "y": {"aggregate": "count", "type": "quantitative"},
        "color": {
          "condition": {
            "selection": "pts",
            "value": "steelblue"
          },
          "value": "grey"
        }
      }
    }
  ],
  "resolve": {
    "legend": {
      "size": "independent"
    }
  },
  "config": {
    "range": {
      "heatmap": {
        "scheme": "greenblue"
      }
    }
  }
}
```

image

**Comment by kanitw** [comment]
I think this should facilitate comparison better?

**Comment by domoritz** [comment]
Size and color are easier to separate. The original example makes it easier to look at the variables in isolation.

**Comment by kanitw** [comment]
Yeah, but it doesn't facilitate comparison at all.  What's the point of overlaying them if you don't want comparison?  In such case, better show only filtered value.

**Comment by domoritz** [comment]
I want comparison but also at the same time see only the highlighted value. See https://www.cs.rpi.edu/~cutler/classes/visualization/F10/papers/ware_texton.pdf

**Comment by kanitw** [comment]
From skimming the screenshot it doesn't seem like your example here exactly match the paper?

What's the goal of this example?  If the goal is to demonstrate that users can create dashboard, I think the uncommon visual encoding design would distract readers from focusing on what it is design to communicate?

File: _data/examples.json [hunk 1]
```diff
@@ -217,6 +217,10 @@
     {
       "name": "interactive_splom",
       "title": "Scatterplot Matrix"
+    },
+    {
+      "name": "concat_layer.vl.json",
+      "title": "Interactive Dashboard"
```
Comment (kanitw) [inline _data/examples.json]: Do you have a name for the upper plot?

"Coordinated bar chart and XXX"? 

(Interactive Dashboard seems pretty generic?)

Comment (domoritz) [reply _data/examples.json]: Cross highlight

**Pull request merged**